homeassistant/components/nissan_leaf/__init__.py

```diff
@@ -287,7 +269,7 @@ def get_next_interval(self):
                 self.data[DATA_PLUGGED_IN] = (
                     server_response.is_connected
                 )
-                async_dispatcher_send(self.hass, SIGNAL_UPDATE_LEAF)
+                dispatcher_send(self.hass, SIGNAL_UPDATE_LEAF)
```

Comment:
We do we send the signal here when we also send the signal at the end of the method? Does it take minutes between this place and the end of the method?

Reply:
Yes! The climate response takes between 30s to 5 minutes to respond, and the same again for the location response.  I have pondered firing the requests to the Nissan servers in parallel, but I don't want to overload both the Nissan servers and the car.

Reply:
Converted to use async_dispatcher_send() here with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/b961a8f38cc6ed433d85542e5110e317a11479ab